metrics-dropwizard/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'scala'
```

Comment:
why not `apply plugin scala`? Also this is also defined in the base build.gradle for all projects so we should drop it from here as it's redundant

Reply:
This is the new way. `apply plugin scala` is the deprecated way.